core/src/core/transaction.rs

```diff
@@ -106,21 +279,24 @@ pub enum KernelFeatures {
 	/// Plain kernel (the default for Grin txs).
 	Plain {
 		/// Plain kernels have fees.
-		fee: u64,
+		#[serde(serialize_with = "fee_fields_as_int")]
+		fee_fields: FeeFields,
```

Comment:
To fully preserve backwards (de)serialization compatibility, we should name this field `fee` instead of `fee_fields

Reply:
renamed in latest commit